js/jquery.pickmeup.js

```diff
@@ -728,7 +728,7 @@
 			options.binded.fill();
 			if ($this.is('input')) {
 				$this
-					.pickmeup('set_date', parseDate($this.val() ? $this.val() : options.default_date, options.format, options.separator, options.locale))
+					.pickmeup('set_date', parseDate($this.val() ? $this.val() : ((options.default_date === true) ? new Date : options.default_date), options.format, options.separator, options.locale))
```

Comment:
I've merged it locally and simplifying some things, I just want to ask why did you add `true` support here? It is not mentioned in modified readme, also since `options.default_date` is `new Date` just by default, why do we need it at all? Looks like might be ignored, right?